src/vcr-sharp/Serializer.cs

```diff
@@ -0,0 +1,129 @@
+﻿namespace VcrSharp
+{
+    using System;
+    using System.Linq;
+    using System.Net.Http;
+    using System.Text;
+    using System.Threading.Tasks;
+
+    public class Serializer
+    {
+        internal static async Task<CachedRequest> Serialize(HttpRequestMessage request)
+        {
+            return new CachedRequest
+            {
+                Headers = request.Headers.ToDictionary(h => h.Key, h => h.Value.ToArray()),
```

Comment:
This includes ApiKey and auth-token values - meaning they are exposed in the generated yaml files. This should be filtered out.

Reply:
This was solved by caching the VCR cassettes files into Travis CI.